src/components/Tabs/Tab.js

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import cx from 'classnames';
+import { getAriaLabel, getTabsClass } from './utils';
+
+const Tab = (props) => {
+  const {
+    children,
+    className,
+    id,
+    isActive,
+    scoped,
+    renderTitle: _,
+    title: __,
+    ...rest
+  } = props;
+
+  return (
+    <div
+      {...rest}
+      aria-labelledby={getAriaLabel(id)}
+      className={cx(
+        getTabsClass('__content', scoped),
+        isActive ? 'slds-show' : 'slds-hide'
+      )}
+      id={id}
+      role="tabpanel"
+    >
+      {isActive === null ? null : children}
+    </div>
+  );
+};
+
+Tab.defaultProps = {
+  className: null,
+  isActive: false,
+  renderTitle: null,
+  scoped: false,
+  tabTitle: null,
```

Comment:
This is not used anywhere, is it? Probably a leftover.

Reply:
right, will fix